vevestaX/vevesta.py

```diff
@@ -1,14 +1,16 @@
+from email.utils import localtime
 import pandas
 import inspect
 import os.path
```

Comment:
you have imported os.path and os both, remove os.path from imports

Reply:
resolved